site/docs/v1/tech/apis/system.adoc

```diff
@@ -442,8 +444,10 @@ The version of the running FusionAuth instance.
 
 This page contains the API that is used for retrieving FusionAuth application metrics to be used with Prometheus. Please refer to the link:/docs/v1/tech/tutorials/prometheus#[Prometheus setup] guide to understand how to set up Prometheus with the FusionAuth metrics endpoint.
 
+By default, this API requires authentication. If you prefer to allow unauthenticated access to this endpoint from local scrapers, you may set `fusionauth-app.local-metrics.enabled=true` as link:/docs/v1/tech/reference/configuration#[documented here].
```

Comment:
We have a lot of these types of links already, if you do a search for `See the link:/docs/v1/tech/reference/configuration` you'll find an existing pattern. 

Can we re-word this to follow that pattern of linking to the configuration reference? Same would go for all new references to this config. 

Also, I don't think you need the trailing `#`.